docs/api/utility-process.md

```diff
@@ -0,0 +1,85 @@
+# UtilityProcess
+
+`UtilityProcess` is the equivalent of [`child_process.fork`][] API from Node.js
+but instead uses [Services API][] from Chromium to create child processes with
+Node.js and Message ports enabled.
```

Comment:
Here I would maybe explain when to use utility processes and when to use node.js forked processes. I.e. advantages and disadvantages (if any). 

For VSCode the main intent why we want this API is to leverage direct renderer<->process communication via message ports.